fetch/api/response/response-static-json.any.js

```diff
@@ -0,0 +1,64 @@
+// META: global=window,worker
+// META: title=Response: json static method
+
+const APPLICATION_JSON = "application/json;charset=utf-8";
+const FOO_BAR = "foo/bar";
+
+const DEFAULT_HEADERS = { "content-type": "application/json;charset=utf-8" };
+const CUSTOM_HEADERS = { "content-type": "foo/bar" };
+const CUSTOM_HEADERS = { "x-foo": "bar" };
+
+const INIT_TESTS = [
+  [undefined, 200, "", APPLICATION_JSON, {}],
+  [{ status: 400 }, 400, "", APPLICATION_JSON, {}],
+  [{ statusText: "foo" }, 200, "foo", APPLICATION_JSON, {}],
+  [{ headers: {} }, 200, "", APPLICATION_JSON, {}],
+  [{ headers: { "content-type": FOO_BAR } }, 200, "", FOO_BAR, {}],
+  [{ headers: { "x-foo": "bar" } }, 200, APPLICATION_JSON, { "x-foo": "bar" }],
```

Comment:
```suggestion
  [{ headers: { "x-foo": "bar" } }, 200, "", APPLICATION_JSON, { "x-foo": "bar" }],
```
status text is missing from this line